src/trackerMiddleware.js

```diff
@@ -232,13 +221,13 @@ export const trackerMiddleware = store => next => action => {
         window.Intercom('update', { last_request_at: parseInt(new Date().getTime() / 1000) })
       }
       break
-    case 'ON_GOOGLE_LOGIN_RETURN':
-      currentUserId = action.payload.googleId
-      const profile = action.payload.profileObj
-      const registerTime =
-        action.payload && action.payload.userReducer && action.payload.userReducer.registerTime
+    case 'POST_LOGIN_PREPARATION_FULFILLED':
+      currentUserId = action.payload.userMetaInfo.googleId
+      const profile = action.payload.userMetaInfo.profile
+      const registerTime = action.payload.userMetaInfo.registerTime
+
       trackUser(currentUserId)
-      intercomLogin(currentUserId, { ...profile.profileObj, registerTime })
+      intercomLogin(currentUserId, { ...profile, registerTime })
```

Comment:
profile is a UserProfileType from backend, which does not contain email.

Reply:
This is different, you can check locally